openedx/core/djangoapps/user_api/helpers.py

```diff
@@ -45,9 +45,20 @@ def _wrapped(*args, **kwargs):
             try:
                 return func(*args, **kwargs)
             except Exception as ex:
-                # Raise the original exception if it's in our list of "ignored" errors
+                # Raise and log the original exception if it's in our list of "ignored" errors
                 for ignored in ignore_errors or []:
                     if isinstance(ex, ignored):
+                        msg = (
+                            u"A handled error occurred when calling '{func_name}' "
+                            u"with arguments '{args}' and keyword arguments '{kwargs}': "
+                            u"{exception}"
+                        ).format(
+                            func_name=func.func_name,
+                            args=args,
+                            kwargs=kwargs,
+                            exception=repr(ex)
+                        )
+                        LOGGER.warning(msg)
```

Comment:
@andy-armstrong do you agree about the "warn" level? I was between "warn" and "info". I don't think it should be "exception" since likely it is just someone calling the API improperly (vs. an internal error).